source/tutorials/php.rst

```diff
@@ -0,0 +1,128 @@
+.. _php:
+
+Installing and Configuring PHP
+******************************
+
```

Comment:
Suggest an opening sentence that defined PHP and PHP-FPM with links to their project pages.
PHP is this.
PHP-FPM s that.

Reply:
Added in latest commit.  Very brief overview, but helpful context.